inference-engine/tests_deprecated/unit/engines/mkldnn/graph/structure/graph_optimization_test.cpp

```diff
@@ -292,7 +291,7 @@ class FakeReLUFactory : public InferenceEngine::ILayerImplFactory {
     InferenceEngine::CNNLayer * cnnLayer;
 };
 
-class FakeFabric : public InferenceEngine::IExtension {
+class FakeFabric : public InferenceEngine::Extensions::Cpu::MKLDNNExtensions {
```

Comment:
I think that tests should be changed. This is fake primitives and in this case we can remove `target_include_directories` on MKLDNN pluign.

Reply:
do you mean to rewrite to new extension mechanism? If yes, it can be done during migration of tests from deprecated folder

Reply:
Yes I mean migration. Ok I got it.